cpp/src/arrow/flight/sql/odbc/tests/connection_test.cc

```diff
@@ -0,0 +1,55 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#ifdef _WIN32
+#include <windows.h>
+//#pragma comment(lib, "odbc32.lib")
+#endif
+
+#include <sql.h>
+#include <sqltypes.h>
+#include <sqlucode.h>
+//#include "arrow/flight/sql/odbc/odbc_api.h"
+//#include "arrow/testing/builder.h"
+#include "gtest/gtest.h"
```

Comment:
```suggestion
#include <sqlucode.h>
#include "gtest/gtest.h"
```
these lines can be removed